Assets/Mirror/Runtime/NetworkConnectionToClient.cs

```diff
@@ -16,6 +16,17 @@ public NetworkConnectionToClient(int networkConnectionId) : base(networkConnecti
         // the client. they would be detected as a message. send messages instead.
         readonly List<int> singleConnectionId = new List<int> { -1 };
 
+        /// <summary>
+        /// Number of seconds of no messages from client after which server will auto-disconnect
+        /// </summary>
+        public float serverIdleTimeout = 30f;
```

Comment:
```suggestion
        public float serverIdleTimeout = 60;
```

a minute seems a bit less stressful here.
just in case of lags, network issues, etc.
it's better to have dead player for 30 more seconds than have someone involuntarily get disconnected, which would be frustrating.